src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/spec/proceed/confirmation/JudgmentSubmittedConfHeader.java

```diff
@@ -24,7 +24,7 @@ public class JudgmentSubmittedConfHeader  implements RespondToResponseConfirmati
     public Optional<String> generateTextFor(CaseData caseData) {
         if ((YesOrNo.NO.equals(caseData.getApplicant1AcceptFullAdmitPaymentPlanSpec()))
             || (YesOrNo.NO.equals(caseData.getApplicant1AcceptPartAdmitPaymentPlanSpec()))
-            && !PAYMENT_PLAN.contains(caseData.getDefenceAdmitPartPaymentTimeRouteRequired())) {
+            || !PAYMENT_PLAN.contains(caseData.getDefenceAdmitPartPaymentTimeRouteRequired())) {
```

Comment:
still not quite what i asked - remember we agreed that we will move this to CaseData boolean method with a readable name so that the code here and below is not repeated and also is more readable